backend/dep/wire.go

```diff
@@ -4,6 +4,7 @@ package dep
 
 import (
 	"database/sql"
+
 	"github.com/short-d/short/app/usecase/risk"
 
```

Comment:
```suggestion
```

Reply:
I think this one was added automatically by `gofmt`.

Reply:
Oh, sorry!

Reply:
@oatovar Let's delete it and do `./scripts/format` again.

Reply:
Thank you it worked!